### PR TITLE
Fixed Barometer Documentation By adding Information and Photos of It

### PR DESCRIPTION
**Pull request by aryanguls**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**
The issue was that for the PSLab documentation site the documentation of some of the instruments provided by PSLab was incomplete and only contained basic headings. 

* **What changes have you introduced?**
For this request I have updated the Barometer documentation by adding information about the instrument , its uses and how it functions on the PSLab app along with relevant screenshots to improve the design of the documentation.

* **Before :** 
Screenshot (440)

* **After :**
Screenshot (441)
Screenshot (442)
Screenshot (443)
Screenshot (444)

### CODE REVIEW

**Comment by aryanguls** [comment]
@mariobehling @hpdang Mentors please review.

File: tutorials/barometer.md [hunk 1]
```diff
@@ -3,3 +3,29 @@ Barometer
...
+1) Android Phone
+2) [PSLab Android App](https://play.google.com/store/apps/details?id=io.pslab&hl=en_US)
+
+Procedure :
+1) Navigate to The PSLab Android App on your device.
+2) Click on the App to turn it on.
+3) Now On the instruments page scroll down until you reach the Barometer instrument option.
+4) On tapping the Barometer instrument the barometer instrument opens and can be used.
+
+![inital setup](../_static/Barometer1.jpg)
```
Comment (NishithP2004) [inline tutorials/barometer.md]: Please reduce size of images.Please refer compass.md file for the required size.I had earlier reduced the sizes of your images in the compass experiment.

**Comment by aryanguls** [comment]
@NishithP2004 thanks for the input. I have resized the images as per the compass.md file. It now looks and fits into the page better.

**Comment by aryanguls** [comment]
Mentors please review as and when possible.

File: tutorials/barometer.md (hunk 1 above)
Comment (NishithP2004) [inline tutorials/barometer.md]: Could you put this image and the following image in a table.
(In order to meet the other tutorials)

**Comment by aryanguls** [comment]
@NishithP2004 I have added the images in the form of a table such that they are one under the other.

**Comment by aryanguls** [comment]
Mentors please review

**Comment by aryanguls** [comment]
@mariobehling @hpdang @CloudyPadmal Sir please accept the merge request. It has already been accepted.

**Comment by cweitat** [comment]
Barometer image to be removed due to no credits given to the owner of the image and unknown sharing license of the image.

**Comment by aryanguls** [comment]
@cweitat The changes have been made and the image has been removed.
Screenshot (497)

**Comment by mariobehling** [comment]
I missed that the images should be moved to the /images folder.

**Comment by aryanguls** [comment]
@mariobehling The images have been added to the /images folder as well.

**Pull request merged**